<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>971</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_liga_desliga">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>780</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botaoOk">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exibição</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Movimentações</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="editaIdade">
          <property name="font">
           <font>
            <family>Arial Rounded MT Bold</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>3</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string notr="true">Digite a Idade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botaoBuscar">
          <property name="text">
           <string>BUSCAR</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="abas_Tab_Widget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_1">
         <attribute name="title">
          <string>Inserção</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="editaNome">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <family>Arial Rounded MT Bold</family>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <underline>false</underline>
                <strikeout>false</strikeout>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="maxLength">
               <number>3</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string notr="true">Digite o Nome</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="botaoInserir">
              <property name="text">
               <string>INSERIR</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Remocao</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="comboRemocao"/>
            </item>
            <item>
             <widget class="QPushButton" name="botaoRemover">
              <property name="text">
               <string>REMOVER</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Manipulações</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLineEdit" name="editaManipulacoes">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>32767</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">Travessias</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Importar</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QComboBox" name="comboImporta">
              <item>
               <property name="text">
                <string>A partir de Texto</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Arquivo texto</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="botaoGerar">
              <property name="text">
               <string>Gerar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Exportar</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QComboBox" name="comboExportar">
              <item>
               <property name="text">
                <string notr="true">Pre-Ordem</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Im-Ordem</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pos-Ordem</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="botaoExporta">
              <property name="text">
               <string>Exportar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_altura_desc">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Altura:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_altura">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>152</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_qtd_nos_desc">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quantidade de Nos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_qtd_nos">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_qtd_folhas_desc">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quantidade de Folhas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_qtd_folhas">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="botaoApagarArvore">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar Arvore</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Travessias</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QLineEdit" name="editaTrav">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>32767</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">Travessias</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboTrav">
        <item>
         <property name="text">
          <string notr="true">Pre-Ordem</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Im-Ordem</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pos-Ordem</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Original</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoMostrarTrav">
        <property name="text">
         <string>Mostrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>graphicsView</zorder>
  <zorder>label_liga_desliga</zorder>
  <zorder>botaoOk</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>botaoApagarArvore</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
